File: Clase8cpp/calculos.py
import sys
import math


if __name__== "__main__":
    i=1
    seno=0
    coseno=0
    tangente=0
    logaritmo=0
    raizCuad=0
    while i<float(sys.argv[1]):
        seno+=math.sin(i)
        coseno+=math.cos(i)
        tangente+=math.tan(i)
        logaritmo+=math.log10(i)
        raizCuad+=math.sqrt(i)
        i=i+i

File: Clase8cpp/calculos.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main(int argc, char *argv[]){
    double n=strtod(argv[1], NULL);
    double i=0;
    double seno=0;
    double coseno=0;
    double tangente=0;
    double logaritmo=0;
    double raiz=0;
    while (i<n)
    {
        seno=sin(i);
        coseno=cos(i);
        tangente=tan(i);
        logaritmo=log(i);
        raiz=sqrt(i);
        i++;
    }

    return 0;
}